packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -11,25 +11,33 @@ import {
     ClipboardCopyVariant
 } from '@patternfly/react-core';
 
-export const UsageDescription = () => (<TextContent>
-    <Text component={ TextVariants.p }>
+export const UsageDescription = () => {
+    React.useEffect(() => {
```

Comment:
No hooks in frontend-components yet. :crying_cat_face: 

Also, I think it would be better to fix it in the Data Driven Forms package, as it is a global issue with the wizard.

Reply:
Dang :sob: I really wanted to use hooks. But that's OK :smiley_cat: maybe next time.
I am not sure where I should do that in DDF. Should we put it as a `"scrollTop": true` attribute in the schema?

Reply:
I think you could open a PR and fix it somewhere around here https://github.com/data-driven-forms/react-forms/blob/620a52ebf0d3513414e80dba34505a02b1185456/packages/pf4-component-mapper/src/form-fields/wizard/wizard-step.js#L14 

or open an issue and I will do it! 

This is really a nice catch!